### ISSUE TITLE
Elasticsearch dashboard

### ISSUE THREAD

**Issue by peycho** [monitoringartist/grafana-aws-cloudwatch-dashboards#63]
Is there Elasticsearch dashboard or I'm missing it somehow?

**Comment by jangaraj** [comment]
No, AWS ES dashboard wasn't published here. Feel free to create pull request.

**Comment by anthonyd-cd** [comment]
hello, is there any way to get AWS ElasticSearch metrics in Grafana at the moment?

**Comment by jangaraj** [comment]
Sure - https://grafana.com/docs/grafana/latest/features/datasources/cloudwatch/

**Comment by anthonyd-cd** [comment]
thanks for the quick reply jangaraj, however I was hoping there it was as easy as choosing the namespace AWS/ElasticSearch as I already do have the Cloudwatch Browser setup but in the namespaces available I see a bunch of services available just not the ElasticSearch

**Comment by jangaraj** [comment]
AWS doc is your good friend, where you should always start https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-managedomains.html#es-managedomains-cloudwatchmetrics

There is no `AWS/ElasticSearch` namespace, but `AWS/ES`.

**Comment by anthonyd-cd** [comment]
Duhh! Thanks Jangaraj! sorry about that, for some reason I associated it to ECS lol Thank you so much! Great work btw

**Comment by strowi** [comment]
for anyone searching.. here is a AWS/ES-Dasboard that seems to work nicely: https://grafana.com/grafana/dashboards/11655
Maybe this can be adapted/copied here (not sure about copyright with dashboards though.. ;))?

**Comment by jangaraj** [comment]
Unfortunately, I can't just copy that dashboard. Author didn't define any license for the dashboard, so it can be a legal problem. I have my own ES dashboard (not published yet), but ES has many metrics, which I didn't add. It will be published in the futue, when I find a time to finish it. Anyway, feel free to create PR.

**Comment by visla-xugeng** [comment]
please let us know when you plan to add es dashboard here. (BTW, AWS migrated to opensearch from elasticsearch)
